File: Sheet #2 (Loops)/C_Even_Odd_Positive_and_Negative.cpp
#include <bits/stdc++.h>

using namespace std;
int main()
{
    int n, ev = 0, odd = 0, po = 0, ne = 0;
    cin >> n;
    int num[n];
    for (int i = 0; i < n; i++)
    {
        cin >> num[i];
        if (num[i] % 2 == 0)
        {
            ev++;
        }
        if (num[i] % 2 != 0)
        {
            odd++;
        }
        if (num[i] > 0)
        {
            po++;
        }
        if (num[i] < 0)
        {
            ne++;
        }
    }
    cout << "Even: " << ev << "\n";
    cout << "Odd: " << odd << "\n";
    cout << "Positive: " << po << "\n";
    cout << "Negative: " << ne << "\n";

    return 0;
}

File: Sheet #2 (Loops)/C_Even_Odd_Positive_and_Negative.java
import java.util.*;

public class C_Even_Odd_Positive_and_Negative {
    public static void main(String args[]) {
        Scanner scan = new Scanner(System.in);

        int t;
        int ev = 0, odd = 0, pos = 0, neg = 0;

        t = scan.nextInt();
        while (t-- > 0) {

            int i = scan.nextInt();

            if (i % 2 == 0) {
                ev++;
            }
            if (i % 2 != 0) {
                odd++;
            }
            if (i > 0) {
                pos++;
            }
            if (i < 0) {
                neg++;
            }
        }
        System.out.println("Even: " + ev);
        System.out.println("Odd: " + odd);
        System.out.println("Positive: " + pos);
        System.out.println("Negative: " + neg);

    }
}